stackimage/README.md

```diff
@@ -0,0 +1,126 @@
+# Application Stack Gradle
```

Comment:
This is nice to add. 

It looks like the heading is wrong though.   This is a more general maven+gradle not just gradle doc right?